pest_derive/Cargo.toml

```diff
@@ -17,7 +17,8 @@ proc-macro = true
 [dependencies]
 quote = "^0.3"
 syn = "^0.10"
-pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
+# pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
```

Comment:
The git dependencies should be kept in order make this easy to depend on. Paths can be use when working locally until we publish on crates.io.